components/fxa-client/src/browser_id.rs

```diff
@@ -42,6 +42,8 @@ impl FirefoxAccount {
             login_state,
             refresh_token: None,
             scoped_keys: HashMap::new(),
+            last_handled_command: None,
+            commands_data: HashMap::new(),
```

Comment:
Perhaps for a follow-up PR, but IIRC the way iOS handles commands is a bit different to the way we do it on Desktop and Android, due to iOS platform restrictions that require it to show UI for every received push notification.  Assuming we intend for iOS to one day use the FxA client component, will the abstractions we've built here allow them to comply with those platform restrictions?